both/api/sources/server/stream-chain/stream-types/removeNullAndUndefinedFields.ts

```diff
@@ -0,0 +1,32 @@
+import { isPlainObject, isArray } from 'lodash';
+
+export function removeNullAndUndefinedFields(something: any): any {
```

Comment:
-2: Undefined fields are not saved into mongo, null fields are intentional this value doesn’t exist assignments. No entrance - entrances: null. This is also documented in a11yjson.

Reply:
Whoops, you're right. Totally forgot about that.